docker/deployment/Makefile

```diff
@@ -34,7 +37,7 @@ deploy-prod: deploy-db deploy-mail-prod deploy-app-prod
 deploy-mail-prod:
 	sed -e s/{{SERVER_HOSTNAME}}/beta.languageforge.org/ mail-deployment.yaml | kubectl apply -f -
 deploy-app-prod:
-  sed -e s/{{WEBSITE}}/beta.languageforge.org/ app-deployment.yaml \
+	sed -e s/{{WEBSITE}}/beta.languageforge.org/ app-deployment.yaml \
```

Comment:
We will want to make sure that after this PR is merged, that our WEBSITE is still languageforge.org as I've done in #1108   This could be a merge conflict once 1108 is merged, I don't know.  Up to you.

Reply:
ok, we'll see

Reply:
i didn't notice that!  Thanks for clarifying this is going into develop.  This is fine.